pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -312,3 +313,13 @@ def progress_bar_metrics(self) -> Dict[str, float]:
             metrics = self.metrics[MetricSource.PBAR]
             self._progress_bar_metrics.update(metrics)
         return self._progress_bar_metrics
+
+    def teardown(self):
+        args = (
+            torch.Tensor,
+            move_data_to_device,
+            "cpu",
```

Comment:
So it gets formatted :)

```suggestion
            "cpu"
```

Reply:
Done !